### ISSUE TITLE
Double checking hyper parameters for 1B post-training

### ISSUE THREAD

**Issue by BrownianNotion** [allenai/open-instruct#1281]
Hi team,
Thanks for adding the post-training configs for OLMo-2-0425 1B in #703! Since these hyperparameters don't appear to be documented in the OLMo-2 paper (please correct me if I'm wrong), I wanted to confirm a few values before I continue.

**A couple of specific questions:**

1. **Total number of episodes in RLVR:** This is set to 2e6 in the command, but in the paper, the 7B/13B models use 1e5/2e5 total episodes respectively- I wonder if this is a typo?
2. **Learning rate discrepancy in RLVR:** The flag `--exp_name 0423_grpo_seed_1_lr_7e-7` suggests a LR of `7e-7`, but the actual value passed is `--learning_rate 5e-7`. Which is correct?
3. **LR scheduler RLVR:** Can you confirm that the 1B model uses a constant LR schedule for GRPO (as opposed to linear with warmup for PPO with the 7B model)?
4. **Other hyperparameters:** Could you also verify that the values for epochs, warmup ratio, and KL beta in the config match what was used for the released model? Though my questions are mainly about the RLVR stage, it'd be good to double check the values across all three stages!
Thank you so much for your time!

**Comment by hamishivi** [comment]
Hi thanks for your interest!
1. For total episodes, we typically would set it to a high number, monitor our in-loop evals, and then stop once we saw scores plateauing/dropping, and release the best checkpoint. Since the 1B was fast, we just ran it through to the end of the 2e6 episodes, but for the larger models we used less episodes since they were slower.
2. 5e-7 is correct, i think the experiment name was just wrong.
3. Yep, we just stick to constant learning rates for our RL experiments right now.
4. From a quick check, the rest of the RL things look right. We generally just copy the commands directly from the jobs that produced the final checkpoints, so they should be correct (and flag values should take precedence over experiment names).
-
